meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -349,3 +349,6 @@ INITRAMFS_IMAGE ?= "flex-initramfs-image"
 INITRAMFS_IMAGE_BUNDLE ?= "${@bb.utils.contains('KERNEL_IMAGETYPES', 'fitImage', '', '1', d)}"
 ## }}}1
 # vim: set fdm=marker fdl=0 :
+
+# Encrypted FS
```

Comment:
This should not be added here. it should go to imx-common or BSP layer. See where we added this in Fir

Reply:
@Noor-Ahsan I made this change because of @ahsanhussain and @abelal comments here:

https://github.com/MentorEmbedded/meta-flex-ref-bsps/pull/124#discussion_r1189860939

Reply:
OK understood

Reply:
This change needs to be moved to the images section in the same file.